### PR TITLE
V2 sys test fix port

### PR DESCRIPTION
**Pull request by jace-roell**
**What It Does**
port system test fixes from V3.

GitHub issue# and link:

#2336
#2353
#2354

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [x] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: packages/cli/__tests__/zostso/__system__/stop/__scripts__/address-space/as_fully_qualified.sh [hunk 1]
```diff
@@ -7,7 +7,8 @@ PORT=$3
 USER=$4
 PASSWORD=$5
 
-SERVLET_KEY=`zowe tso start as --host $HOST --account $ACCOUNT --port $PORT --user $USER --password $PASSWORD --ru=false | grep -oP "(?<=: ).*"`
+# SERVLET_KEY=`zowe tso start as --host $HOST --account $ACCOUNT --port $PORT --user $USER --password $PASSWORD --ru=false | grep -oP "(?<=: ).*"`
```
Comment (gejohnston) [inline packages/cli/__tests__/zostso/__system__/stop/__scripts__/address-space/as_fully_qualified.sh:10]: Is there a reason why we are keeping the original statement around as a comment in this file and the next 2 files?

**Review by awharn — APPROVED** [review]
Everything works great! Thanks for backporting these, @jace-roell!

File: jest.config.js
Comment (t1m0thyj) [inline jest.config.js:1]: Please remove lines 141-144, as they duplicate the `jest-stare` config which needs to live in package.json instead 😋

**Review by gejohnston — APPROVED** [review]

**Review by traeok — APPROVED** [review]
LGTM, thanks Jace for porting this over!

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋 
---

From a little earlier this morning!
```
Test Suites: 138 passed, 138 total
Tests:       3 skipped, 1201 passed, 1204 total
Snapshots:   100 passed, 100 total
Time:        2814.569 s
```

**Pull request merged**